src/Adapter/Product/Combination/Update/CombinationStockUpdater.php

```diff
@@ -66,11 +75,13 @@ class CombinationStockUpdater
     public function __construct(
         StockAvailableRepository $stockAvailableRepository,
         CombinationRepository $combinationRepository,
-        StockManager $stockManager
+        StockManager $stockManager,
+        Configuration $configuration
```

Comment:
Use configuration interface

Reply:
I can't because I need to use the ShopContraint parameter which is not in the interface yet.